glibc-2.28 did hide loff_t indef system-specific macros:
    https://sourceware.org/git/?p=glibc.git;a=commitdiff;h=663e7d784977f6b15c0cab73f754f9f39c9c0c2c

This caused build failure:
    ../include/reiserfs_lib.h:300:55: error: unknown type name 'loff_t'; did you mean 'off_t'?

Re-enable extensions to pull loff_t back.

https://bugs.gentoo.org/663930
--- a/configure.ac
+++ b/configure.ac
@@ -23,2 +23,4 @@ AC_PROG_MAKE_SET
 AC_PROG_LIBTOOL
+dnl pull in loff_t from glibc
+AC_USE_SYSTEM_EXTENSIONS
 
